docs/pages/ktexttruncator.vue

```diff
@@ -0,0 +1,20 @@
+
+import KTextTruncator from '../../lib/KTextTruncator.vue';
```

Comment:
This import is not needed, I believe (and if it was, it would need to be done in the `<script>` part of the Vue file